sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -360,6 +360,7 @@ trait CheckAnalysis extends PredicateHelper {
 
           case InsertIntoTable(t, _, _, _, _)
```

Comment:
cc @yhuai do you remember why we have this check? `InsertIntoTable` can only be used for table right? When will we hit this branch?